<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuWindow</class>
 <widget class="QMainWindow" name="MenuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Label_Mail">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Example@mail.ru</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="LogOut_button">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Выйти</string>
    </property>
   </widget>
   <widget class="QPushButton" name="History_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>История операций</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Budget_button">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Бюджеты</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Settings_button">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>90</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Настройки</string>
    </property>
   </widget>
   <widget class="QLabel" name="Remind_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Предстоящие платежи:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Payments_button">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>90</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Предстоящие платежи</string>
    </property>
   </widget>
   <widget class="QLabel" name="Name_Window_MenuWindow">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>141</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Imprint MT Shadow</family>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Menu</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="NullRemind_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>771</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>У Вас нет предстоящих платежей</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
